### PR TITLE
Portfolio: Projects Feature

### PR DESCRIPTION
**Pull request by ZikaZaki**
### In this pull-request I:
 - [x] Created the **Works** section
 - [x] Created the **About myself** section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -69,6 +78,242 @@ <h1 class="primary-txt">
...
+                        </button>
+                     </div>
+                </div>
+            </div>
+        </section>
+        <section class="about-myself">
+
+        </section>
+        <section class="contact">
+        </section>
```
Comment (isaacpitwa) [inline index.html]: - [x] it is a requirement for this project to include the contact section. Kindly complete this section.

File: assets/css/index.css [hunk 2]
```diff
@@ -1,128 +1,244 @@
...
-svg {
-  margin-right: 10px;
+.connect-icons ul li {
+  display: inline;
 }
 
-.navbar {
+.works {
+  position: absolute;
+}
```
Comment (isaacpitwa) [inline assets/css/index.css]: - [x] it is required for this project. to complete this section using a grid layout. Kindly change the styles to achieve the same design using grid layout.

File: index.html [hunk 3]
```diff
@@ -69,6 +78,242 @@ <h1 class="primary-txt">
...
         </section>
+        <section class="works">
+            <div class="card-works">
+                <div class="snapshoot-portfolio bg-1">
+                </div>
+                <div class="left-block">
+                    <div>
+                        <h1 class="primary-txt">
+                            Tonic
+                        </h1>
```
Comment (isaacpitwa) [inline index.html]: - [x] For proper semantics and improved SEO, it is advised to use one single h1 per web page to emphasize only one most important heading. kindly change the work card title to a `<h2....h6>` tag to also reduce spamming. (having multiple h1) in a webpage

File: index.html [hunk 4]
```diff
@@ -69,6 +78,242 @@ <h1 class="primary-txt">
...
+                            <p>
+                                <span class="tag">css</span>
+                            </p>
+                        </div>
+                        <div>
+                            <p>
+                                <span class="tag">javascript</span>
+                            </p>
+                        </div>
+                    </div>
```
Comment (isaacpitwa) [inline index.html]: - [x] In order to improve sematics, kindly make use of the `ul` and `li` tags in this section.

**Review by isaacpitwa — CHANGES_REQUESTED** [review]
Hi @ZikaZaki 🙂 ,

You have done a tremendous job setting up your project. I really like it :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text


## Highlights 
- All linters are passing  ✅
- Correct use of GitHub flow ✅
- Well documented Repository ✅
-  Clean and readable code base ✅

## Required changes ♻️ 

Check the comments under the review.

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Check the comments under the review.





Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @isaacpitwa.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (ZikaZaki) [reply index.html]: Hi @isaacpitwa  🙂 ,
I hope you are doing well,

In this project, we are only asked to build the middle section of the mobile website. Hence, the **contact** section was not included in my last submission. Here's a link to the **[project-requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)**. However, thanks for the great feedback and for your time and cooperation.

**Pull request merged**